packages/typegpu/src/data/index.ts

```diff
@@ -2,12 +2,20 @@
  * @module typegpu/data
  */
 
-export { bool, f32, f16, i32, u32 } from './numeric';
+export {
+  bool,
+  f32,
+  f16,
+  i32,
+  u32,
+  abstractInt,
+  abstractFloat,
```

Comment:
Is there a need for `abstractInt` and `abstractFloat` to be part of the public API, or not? From what I understand, WGSL essentially hides them, and does not allow their use in user-defined shaders.